docs/docs/manual/exporting.md

```diff
@@ -6,34 +6,97 @@ sidebar_label: Exporting
+
+### Templating exporter
+
+If you pick <span class="menuItems">Templating…</span> from the <span class="menuItems">Export</span> dropdown menu, you can “roll your own” exporter. This is useful for formats that we don't support natively yet, or won't support. The Templating exporter generates JSON by default. 
+
+The window that appears allows you to set your own separators, prefix, and suffix to create a complete dataset in the language of your choice. In the <span class="menuItems">Row Template</span> section, you can choose which columns to generate from each row by calling them with variables. 
+
+This can be used to:
+* output reconciliation data (`cells["column name"].recon.match.name`, `.recon.match.id`, and `.recon.best.name`, for example) instead of cell values
+* create multiple columns of output from different member fields of a single project column
+* employ GREL expressions to modify cell data for output (for example, `cells["column name"].value.toUppercase()`). 
+
+Anything that appears inside curly braces ({{}}) is treated as a GREL expression; anything outside is generated as straight text. You can include [regular expressions](expressions#regular-expressions) as usual (inside forward slashes, with any GREL function that accepts them), with the exception that you cannot use curly braces (for example, to denote a `{min,max}` capture range). 
+
+You could also simply output a plain-text document inserting data from your project into sentences (for example, "In `{{cells["Year"].value}}` we received `{{cells["RequestCount"].value}}` requests.").
+
+![A screenshot of the Templating exporter generating JSON by default.](/img/templating-exporter.png)
+
+Once you have created your template, you may wish to save the text you produced in each field, in order to reuse it in the future. Once you click “Export” OpenRefine will output a simple text file, and your template will be discarded.
+
+We have recipes on using the Templating exporter to [produce several different formats](https://github.com/OpenRefine/OpenRefine/wiki/Recipes#12-templating-exporter).
 
 ## Export a project
 
+You can share a project in progress with another computer, a colleague, or with someone who wants to check your history. This can be useful for showing that your data cleanup didn’t distort or manipulate the information in any way. Once you have exported a project, another OpenRefine installation can [import it as a new project](starting#import-a-project). 
+
+From the <span class="menuItems">Export</span> dropdown, select <span class="menuItems">OpenRefine project archive to file</span>. OpenRefine exports your full project with all of its history. It does not export any current views or applied facets. Any reconciliation information will be preserved, but the importing installation will need to add the same reconciliation services to keep working with that data. 
```

Comment:
Is it worth us emphasising here that the fact it is the entire project means that any recipient can step the project back to its original state - so, for example, if there was confidential information in the original data import that you don't want to share, you shouldn't share the project file directly even if you removed that data while working with the project?

Reply:
Yes, absolutely emphasize.  That point of "OpenRefine Projects contain confidential data from previous steps which is hidden but still accessible to anyone who has the OpenRefine Project file" is something that I emphasize when I give trainings.  "When you don't want to expose any history or hidden, removed, confidential data, but only the current selected data as sharable, then export in any other format besides an OpenRefine Project".